src/javascript/app/pages/trade/markets.jsx

```diff
@@ -536,11 +538,12 @@ class Markets extends React.Component {
                                             >
                                                 <div
                                                     className={classNames('market', {
-                                                        'active': subgroup_active,
+                                                        'active'         : subgroup_active,
+                                                        'accordion-label': !!open_accordion,
                                                     })}
-                                                    onClick={toggleAccordion || (subgroup_active ? toggleAccordion : '')}
+                                                    onClick={toggleAccordion}
                                                 >
-                                                    <span className={`icon synthetic_index ${open_accordion ? 'active' : ''}`} />
+                                                    <span className={`icon synthetic_index ${subgroup_active ? 'active' : ''}`} />
```

Comment:
cant we write this similar to line 541 using classNames function

Reply:
Resooolved 😩😩😩